skins/oasis/js/privacyUpdateNotification.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
+
+    function getCookie(name) {
+        var nameEQ = name + '=';
+        var ca = document.cookie.split(';');
+    
+        for (var i = 0; i < ca.length; i++) {
+            var c = ca[i];
+            while (c.charAt(0) === ' ') c = c.substring(1, c.length);
+            if (c.indexOf(nameEQ) === 0) return c.substring(nameEQ.length, c.length);
+        }
+    
+        return null;
+    }
+
+    function setCookie(name) {
```

Comment:
How about using `window.Wikia.Cookies.set()` instead?

Reply:
I like it!